submissions/Idayatolaide/CSS TASK1/HTML.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
Nice work. Please remove the space in folder name

Reply:
I have removed the spacing

Reply:
I newly added my landing page

Reply:
Well done.  Please add a style to the list tags, they are not showing when viewed in the browser.